src/plugins/intel_cpu/src/nodes/transpose.cpp

```diff
@@ -174,6 +206,55 @@ void MKLDNNTransposeNode::createPrimitive() {
     }
 }
 
+void MKLDNNTransposeNode::createReorderPrimitive(const mkldnn::memory::desc& desc, void* srcPtr, void* dstPtr, bool isInput) {
```

Comment:
Not required for this PR. but we seems like it worth to make createReorderPrimitive static method of Reorder node to avoid copy-paste.